hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/TransactionProcessor.java

```diff
@@ -209,6 +215,7 @@ private InvolvedParties computeInvolvedParties(
                 final var alias = transaction.contractIdOrThrow().evmAddressOrThrow();
                 validateTrue(isEvmAddress(alias), INVALID_CONTRACT_ID);
                 parties = new InvolvedParties(sender, relayer, pbjToBesuAddress(alias));
+                updater.setupTopLevelLazyCreate(parties.receiverAddress);
```

Comment:
I don't see where we actually make use of the `PendingCreate`.  Is that yet to come?

Reply:
It's used [here](https://github.com/hashgraph/hedera-services/blob/07871-complete-assorted-xtest/hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/state/ProxyWorldUpdater.java#L303) to make `ProxyWorldUpdater.createAccount()` work as expected.

Reply:
thanks